lib/phoenix_live_view/js.ex

```diff
@@ -889,16 +889,24 @@ defmodule Phoenix.LiveView.JS do
   end
 
   defp put_op(%JS{ops: ops} = js, kind, args) do
+    args = drop_nil_values(args)
     %JS{js | ops: ops ++ [[kind, args]]}
   end
 
+  defp drop_nil_values(args) when is_map(args) do
+    Enum.reject(args, fn {_k, v} -> is_nil(v) end)
```

Comment:
Since we are traversing the map here, I would pass it as a keyword list, so we avoid unnecessary conversions!

Reply:
We would still return the map though.